test/onnx/gen_onnx.py

```diff
@@ -1014,9 +1038,9 @@ def const_of_shape_int64_test():
 def const_of_shape_no_value_attr_test():
     shape_val = np.array([2, 3, 4]).astype(np.int64)
     shape_ts = helper.make_tensor(name='shape_tensor',
-                                  data_type=TensorProto.INT32,
+                                  data_type=TensorProto.INT64,
```

Comment:
Why change the protobuf on these instead of making a new test with the different datatype?

Reply:
Not sure I understand your question, I changed this datatype because the ONNX spec says it must be a int64 type

Reply:
Ah so correctness, that makes sense. All good!